### ISSUE TITLE
Switch Figma Links in Storybook to Release File

### ISSUE THREAD

**Issue by thrbnhrtmnn** [deven-org/B01LER-Kitchen#726]
#### Description / User Story

Currently all links to Figma components in Storybook (in the Design Addon) are linking to the B01LER Working File and not to the B01LER Release File. All links have to be changed.

For future releases it would be nice to have something in place to automatically change all the links to the newest Figma File.

As discussed in 04.01.2024, we will get rid of the current release process for figma files with #554 once the token renaming is done for all component tokens. The link to the release file will still be a different one and we have to check, of the link will change with the releases. If yes, we should also have a mechanism to do this automatically for following releases.

----
#### Requirements
- [ ] #554 

----
#### Acceptance Criteria
- [ ] Links to Figma within Storybook have been changed to the public File (current release)
- [ ] If link to the current Figma file changes with a new release, we have an easy way to change all Figma links within Storybook to a new File

----
#### Background information
- [Working File](https://www.figma.com/file/C4vgEKz8mKyulJ4gm3Qdql/%F0%9F%AB%A7-%5BBLR%5D-The-B01LER?type=design&node-id=701%3A1782&mode=design&t=oNaVLaYTyQUkvvjY-1)
- [Release File](https://www.figma.com/file/zPHgVJFeLAhdM4MdOymhPb/B01LER--%5Bv0.1---alpha-release%5D?type=design&node-id=701%3A1782&mode=design&t=NGoPoZHcsXWb7R3q-1)

**Comment by thrbnhrtmnn** [comment]
This issue will be done within #554
